templates/default/convert-ipython-notebook.sh.erb

```diff
@@ -54,6 +54,14 @@ if [ $? -ne 0 ] ; then
     exit 6
 fi    
 
+
+echo "
+from pyspark import SparkContext
+sc = SparkContext(\"yarn\", \"${base}\")
+
+" > /tmp/"${base}".py
+cat "${base}".py >> /tmp/"${base}".py
+
```

Comment:
According to @robzor92  this not correct. The PySpark kernel creates a `SparkSession` object named `spark` not a `SparkContext` named sc.

Reply:
I am not opposed the idea. But as it is right now it is wrong if we want to make it compatible with Jupyter. It is as Fabio said. We should not get the **SparkContext**, but the **SparkSession**. And it **must** be named **spark**. You get the SparkSession like this. https://github.com/logicalclocks/hops-util-py/blob/7df48d2ecc6153513f6aed097335a16fa49b4ca9/hops/util.py#L408

Reply:
I am also assuming this is not tested. So unless it is done so properly after the changes, please remove this code. I will reintroduce it in HOPSWORKS-716 in that case.

Reply:
I removed this part of code